source/admin_manual/list_of_events.rst

```diff
@@ -561,8 +561,8 @@ These events apply only for connections using the ``connection API``.
 .. dovecot_core:event:: client_connection_disconnected
    :inherit: client_connection_common
 
-   :field bytes_in: Amount of data read, in bytes.
-   :field bytes_out: Amount of data written, in bytes.
+   :field in_bytes: Amount of data read, in bytes.
+   :field out_bytes: Amount of data written, in bytes.
```

Comment:
Probably would be good to have `.. changedsince:: v2.4.0;v3.0.0 Field was renamed from bytes_in` although kind of annoying to add that to every one of these fields.

In any case this is missing from upgrading page.